test/functional/api/cas/casadm_parser.py

```diff
@@ -190,7 +190,7 @@ def get_flush_parameters_alru(cache_id: int):
 
 def get_flush_parameters_acp(cache_id: int):
     casadm_output = casadm.get_param_cleaning_acp(cache_id,
-                                                  casadm.OutputFormat.csv).stdout.spltlines()
+                                                  casadm.OutputFormat.csv).stdout.splitlines()
```

Comment:
Also fixed in https://github.com/Open-CAS/open-cas-linux/pull/230 PR and it's not only typo here.